cinema/views.py

```diff
@@ -127,6 +122,19 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "genres",
+                type={"type": "array", "items": {"type": "number"}},
+                description="Filter by genres id (ex. ?genres=2,3)",
+            )
+        ]
+    )
```

Comment:
what about other params ?
![image](https://github.com/mate-academy/py-documentation-and-tests/assets/103201010/a9efc2e3-40ac-4357-a355-13de452ed045)